tests/multi_sig.rs

```diff
@@ -98,8 +98,8 @@ fn test_proof_multisig(
     wallets.iter().enumerate().for_each(|(i, wallet)| {
         let balance = wallet.get_balance().unwrap();
         assert!(
-            (4_999_999_256..=4_999_999_596).contains(&balance.confirmed),
-            "balance of wallet {} is {} but should be between 4'999'999'256 and 4'999'999'596",
+            (49_999_999_256..=49_999_999_596).contains(&balance.confirmed),
+            "balance of wallet {} is {} but should be between 49'999'999'256 and 49'999'999'596",
```

Comment:
⛏️ Since different countries use different 1000's separators can we use the rust standard for messages here and below?
```suggestion
            "balance of wallet {} is {} but should be between 49_999_999_256 and 49_999_999_596",
```